pkgs/by-name/re/regolith/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "regolith";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "Bedrock-OSS";
+    repo = pname;
+    tag = version;
+    hash = "sha256-gTEQ2hu581tD1I/3iLHzE/2nekAG49/M6V6QeqPhYsA=";
+  };
+
+  doCheck = false;
```

Comment:
Why? Are the tests failing, and is there nothing to be done? Plus, have you reported the potential test failures to the upstream?

At minimal, this will need a comment explaining why the package tests fail and when it will be safe to uncomment.

Reply:
They require network connection. I added a comment.